website/docs/help-and-support/troubleshooting-guide/deployment-errors.mdx

```diff
@@ -125,6 +125,31 @@ When manually updating Appsmith without using the auto-update system, old and un
 
 After each manual update, run `docker system prune -a` to clean up unused Docker images, containers, volumes, and networks. This will free up disk space and prevent similar issues in the future.
 
+# Appsmith unable to connect to azure postgreSQL database
+
+Appsmith, hosted on Azure Web App, cannot connect to a PostgreSQL database (Azure Database for PostgreSQL flexible server).
+
+
+#### Error message
+
+<Message
+  messageContainerClassName="error"
+  messageContent="Test failed, couldn't establish a connection"
+></Message>
+
+
+#### Cause
+
+The Appsmith instance is not integrated into the same Azure virtual network (vnet) as the PostgreSQL database. As a result, the two services are unable to communicate because they are on different networks.
+
+#### Solution
+
+- **Integrate Appsmith into the Azure Virtual Network (vnet):** Ensure that your Appsmith instance is part of the same Azure virtual network as your PostgreSQL database.
```

Comment:
Follow the same style as we do for all other solutions. The heading for each solution is not needed. Also, the tone should be to help the user verify. Rewrite in that format.